scripts/check_style.sh

```diff
@@ -20,6 +20,11 @@ EXCLUDE_FILES=(
 EXCLUDE_FILES_JOINED=$(printf "%s\|" "${EXCLUDE_FILES[@]}")
 EXCLUDE_FILES_JOINED=${EXCLUDE_FILES_JOINED%??}
 
+INCLUDE_DIRECTORIES=(
+    "libevmasm"
+)
+INCLUDE_DIRECTORIES_JOINED=$(printf "%s/* " "${INCLUDE_DIRECTORIES[@]}")
```

Comment:
Not that it matters much, but the `_JOINED` thing was here mostly because we wanted the `|` *between* items. In your case you just append `/*` so just a simple array is enough:

```bash
NAMESPACE_STD_FREE_FILES=(
    "libevmasm/*"
)
```

```bash
grep -nIE -d skip "using namespace std;" "${NAMESPACE_STD_FREE_FILES[@]}"
```

Incidentally, having the array used directly and quoted would also solve [the error you're getting in `check_style` right now](https://app.circleci.com/pipelines/github/ethereum/solidity/30480/workflows/e4b2bb75-8182-4947-98c8-15f6add6d728/jobs/1352542).

Reply:
Hmmm, the issue is that if I string quote `"${NAMESPACE_STD_FREE_FILES[@]}", it'll yield
```
grep -nIE -d skip 'using namespace std;' 'libevmasm/*'
```
and paths seemingly cannot be strings, since it'll fail with the following error:
```
grep: libevmasm/*: No such file or directory
```
I would then have to omit the quotes, i.e. `${NAMESPACE_STD_FREE_FILES[@]}`, but that will then cause the error I get now:
```
SC2086 (info): Double quote to prevent globbing and word splitting.
```
And word splitting and globbing is exactly what I want here, so not sure how to proceed.

Reply:
Doesn't work either, I'll look into disabling the warning.